src/main/java/core/basesyntax/FibonacciSequence.java

```diff
@@ -10,6 +10,17 @@
      * Результат: 21</p>
      */
     public int getFibonacciNumber(int index) {
-        return 0;
+        int numbrer0 = 1;
+        int numbrer1 = 1;
+        int numbrer2 = 0;
+        if (index == 0) {
+            return 0;
+        }
+        for (int i = 3; i <= index; i++) {
```

Comment:
basically, this task is a classic task for recursion. Please replace for loop with recursion. It would be the best option

Reply:
yes, you are right! I changed solution. used recursion.